### ISSUE TITLE
Are the stems in the inception resnet v2 paper different from that in the code here?

### ISSUE THREAD

**Issue by yxchng** [tensorflow/models#1235]
<img width="430" alt="screen shot 2017-03-22 at 10 25 34 am" src="https://cloud.githubusercontent.com/assets/10518587/24179422/f705c0de-0ee9-11e7-84ff-5e841ebfe043.png">
<img width="669" alt="screen shot 2017-03-22 at 10 25 20 am" src="https://cloud.githubusercontent.com/assets/10518587/24179425/fb58abce-0ee9-11e7-9af2-156ed65b71ad.png">

Here it seems that the stem is only one branch but the paper has 2 branches.

**Comment by yxchng** [comment]
Even if I just take the left branch, the numbers also do not match.

**Comment by yxchng** [comment]
Also can I know where are batch norm added in inception resnet v2?

**Comment by aselle** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by hamddan4** [comment]
I couldn't found why they implemented the Stem block of InceptionResnetV1 instead of InceptionResnetV2' for InceptionResnetV2 either. Here's the question I've asked, but nobody answered yet: https://stackoverflow.com/questions/64488034/inceptionresnetv2-stem-block-keras-implementation-mismatch-the-one-in-the-origin

**Comment by hamddan4** [comment]
Just found what happened: https://stackoverflow.com/questions/64488034/inceptionresnetv2-stem-block-keras-implementation-mismatch-the-one-in-the-origin
